src/TraceEvent/EventPipe/RewindableStream.cs

```diff
@@ -0,0 +1,167 @@
+﻿using PEFile;
```

Comment:
I think this is unused?

Reply:
yeah, it should be unused. I think VS's automatic adding of namespaces must have gotten a little over-zealous and I failed to notice :)

Reply:
Happens to me all the time.  I'm tempted to do a repo-wide pass.  I'm afraid of how many of these would show up.  :)